### PR TITLE
interfaces/builtin/fwupd.go: tag missing devices

### PR DESCRIPTION
**Pull request by valentindavid**
Also:

interfaces/builtin/fwupd.go: allow locking for ipmi device nodes

### CODE REVIEW

File: interfaces/builtin/fwupd.go
Comment (Meulengracht) [inline interfaces/builtin/fwupd.go:391]: Why are these removed?

**Review by Meulengracht — APPROVED** [review]
LGTM, however I think some more context/description of why we are removing tagging from tpm/nvme and replacing it with i2c and ipmi would be nice

File: interfaces/builtin/fwupd.go
Comment (bboozzoo) [reply interfaces/builtin/fwupd.go:391]: the list was sorted, tpm is few lines down

File: interfaces/builtin/fwupd.go
Comment (bboozzoo) [inline interfaces/builtin/fwupd.go:397]: TPM resource manager is new thing here, IDK if it needs an ack from @alexmurray

**Review by bboozzoo — APPROVED** [review]
LGTM

File: interfaces/builtin/fwupd.go
Comment (valentindavid) [reply interfaces/builtin/fwupd.go:397]: Yes, I had marked as "Needs security review". I have requested is review now.

This is new for the cgroup, but it was already the apparmor profile.

**Review by alexmurray — APPROVED** [review]
LGTM. Thanks.

File: interfaces/builtin/fwupd.go
Comment (alexmurray) [reply interfaces/builtin/fwupd.go:397]: tpmrm doesn't provide any additional risk compared to directly accessing tpm devices so there is no security risk here.

File: interfaces/builtin/fwupd.go
Comment (Meulengracht) [reply interfaces/builtin/fwupd.go:391]: Ah okay

**Comment by superm1** [comment]
Can this please merge? We have another issue raised that should fix by it:

https://github.com/fwupd/fwupd/issues/6725

**Pull request merged**
